pkg/kepctl/query.go

```diff
@@ -111,6 +146,19 @@ func sliceToMap(s []string) map[string]bool {
 	return m
 }
 
+func sliceContains(s []string, e string) bool {
+	found := false
+
+	for _, k := range s {
+		if k == e {
+			found = true
```

Comment:
just return true here, return false at the end. no need for the `found` variable

Reply:
Done.